features/onboarding/views/forms/_ci_bot_account.erb

```diff
@@ -0,0 +1,66 @@
+<!-- TODO: setup bot permisisons for the repo, etc. -->
+
+<form id="ci_bot_account" method="POST" action="/onboarding/ci_bot_account">
+  <h4>2. CI Bot Account</h4>
+
+  <p>
+    <code>fastlane.ci</code> requires you setup a CI <i>bot account</i> to
```

Comment:
Any help with this wording would be greatly appreciated. LOL :joy:

Reply:
I'd really use something like [this](http://danger.systems/guides/getting_started.html#creating-a-bot-account-for-danger-to-use)

Reply:
Thank you!! I'll update the documentation to be more like this page.